src/styles/widget.scss

```diff
@@ -7,7 +7,7 @@
   position: relative;
   filter: drop-shadow(0 1px 5px rgba(0, 0, 0, 0.1));
 
-  @media (min-width: 430px) {
+  &[data-size-large='true'] {
```

Comment:
This is how we'll be using `data-size-large` when it is set to true/false in `WidgetContent.vue`
![image](https://user-images.githubusercontent.com/63985172/136717016-b40fd770-eeb6-4813-b439-60e23ffda3f7.png)